src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -281,6 +281,19 @@ def __write_kv_and_features_to_file(file_path, key_values=None, features=None, f
         raise CLIError("Failed to export key-values to file. " + str(exception))
 
 
+# Exported in the format @Microsoft.AppConfiguration(Endpoint=<storeEndpoint>; Key=<kvKey>; Value=<kvValue>; Label=<kvLabel>).
```

Comment:
We shouldn't export any 'Value' in the reference

Reply:
I see you're not really adding value in the reference. Can you update the comment?

Reply:
Thanks for pointing this out. pushed an update to address this.